File: Array4.c
#include<stdio.h>

int main()
{
   int Arr[5] = {10,20,30,40,50};
   
   printf("base address of array:%d\n",Arr);
   printf("First element is:%d\n",&(Arr[0]));
   printf("First element is :%d\n",Arr[0]);
   printf ("size of whole Array :%d\n",sizeof(Arr));
   printf("size of second element of array:%d\n",(Arr[1]));

    return 0;
}

File: Array4.java
class Array4
{
    public static void main(String Arg[])
    {
      int Arr[][]=new int[4][];

      Arr[0]= new int[2];
      Arr[1]= new int[5];
      Arr[2]= new int[3];
      Arr[3]= new int[7];

      
        int i=0,j=0;
        for (i =0;i<Arr.length;i++)
        {
            for(j=0;j<Arr[i].length;i++)
            {
            System.out.print(Arr[i][j]+"\t");
            }
            System.out.println();
        }
    }
}